modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -35,6 +35,38 @@ Perform the migration only when an interruption in service is acceptable.
 $ oc get Network.config.openshift.io cluster -o yaml > cluster-openshift-sdn.yaml
 ----
 
+. Verify that the  `OVN_SDN_MIGRATION_TIMEOUT` environment variable is set and is equal to `0s` by running the following command:
++
+[source,bash]
+----
+#!/bin/bash
+
+if [ -n "$OVN_SDN_MIGRATION_TIMEOUT" ] && [ "$OVN_SDN_MIGRATION_TIMEOUT" = "0s" ]; then
+    unset OVN_SDN_MIGRATION_TIMEOUT
+fi
```

Comment:
Should this be `if`?

Reply:
That looks right to me. It closes the if statement. I'm not an engr though so I'll double check.